tests/FakeItEasy.IntegrationTests.External.netstd/project.json

```diff
@@ -0,0 +1,34 @@
+﻿{
+  "name": "FakeItEasy.IntegrationTests.External",
+  "version": "1.0.0-*",
+  "description": "FakeItEasy.IntegrationTests.External.netstd Class Library",
+
+  "buildOptions": {
+    "compile": {
+      "include": [
+        "../FakeItEasy.IntegrationTests.External/**/*.cs"
+      ],
+      "exclude": [
+      ]
+    }
+  },
+
+  "dependencies": {
+    "NETStandard.Library": "1.6.0",
+    "FakeItEasy": {
+      "target": "project",
+      "version": "99.99.99-wrapped"
+    }
+  },
+
+  "frameworks": {
+    "netstandard1.6": {
+      "imports": "dnxcore50"

```

Comment:
I've never understood the "imports" property fully, and it probably does no harm, but I think it can probably be removed.